src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -76,28 +94,65 @@ public void shouldNotSendHwfSuccessfulConfirmationEmail() throws Exception {
     }
 
     @Test
-    public void sendAssignToJudgeConfirmationEmail() throws Exception {
+    public void sendAssignToJudgeConfirmationEmailIfDigitalCase() throws Exception {
         buildCcdRequest(CCD_REQUEST_WITH_SOL_EMAIL_CONSENT_JSON);
         mockMvc.perform(post(ASSIGN_TO_JUDGE_CALLBACK_URL)
-                .content(requestContent.toString())
-                .contentType(MediaType.APPLICATION_JSON_VALUE))
-                .andExpect(status().isOk());
+            .header(AUTHORIZATION_HEADER, AUTH_TOKEN)
+            .content(requestContent.toString())
+            .contentType(MediaType.APPLICATION_JSON_VALUE))
+            .andExpect(status().isOk());
 
         verify(notificationService, times(1))
                 .sendAssignToJudgeConfirmationEmail(any(CallbackRequest.class));
+        verifyNoInteractions(assignedToJudgeDocumentService);
+        verifyNoInteractions(bulkPrintService);
     }
 
     @Test
     public void shouldNotSendAssignToJudgeConfirmationEmail() throws Exception {
         buildCcdRequest(CCD_REQUEST_JSON);
         mockMvc.perform(post(ASSIGN_TO_JUDGE_CALLBACK_URL)
-                .content(requestContent.toString())
-                .contentType(MediaType.APPLICATION_JSON_VALUE))
-                .andExpect(status().isOk());
+            .header(AUTHORIZATION_HEADER, AUTH_TOKEN)
+            .content(requestContent.toString())
+            .contentType(MediaType.APPLICATION_JSON_VALUE))
+            .andExpect(status().isOk());
 
         verifyNoMoreInteractions(notificationService);
     }
 
+    @Test
+    public void sendAssignToJudgeNotificationLetterIfIsConsentedAndIsPaperApplication_AndToggledOn() throws Exception {
+        buildCcdRequest(BULK_PRINT_PAPER_APPLICATION_JSON);
+
+        when(featureToggleService.isAssignedToJudgeNotificationLetterEnabled()).thenReturn(true);
```

Comment:
When enabling this toggle in prod, are we going to delete this test?

Reply:
No we'll keep the test - but just remove the 'andToggledOn' part and remove the line:
' when(featureToggleService.isAssignedToJudgeNotificationLetterEnabled()).thenReturn(true);'

Reply:
if its a mocked unit test surely it wouldn't cause any issues in prod?

Reply:
Yeah it'll not cause any issues in Prod because of the mock but just for code cleanliness we should remove the 'andToggledOn' part - nothing functionality wise should change